compiler+runtime/include/cpp/jank/runtime/core/make_box.hpp

```diff
@@ -60,6 +60,12 @@ namespace jank::runtime
     return make_box<runtime::obj::real>(r);
   }
 
+  [[gnu::always_inline, gnu::flatten, gnu::hot]]
+  inline auto make_box(obj::ratio_data const r)
+  {
+    return expect_object<runtime::obj::ratio>(obj::ratio::create(r.numerator, r.denominator));
```

Comment:
So, when we construct with a `ratio_data`, we know for certain that `ratio::create` will always give us back a ratio? Is there any possibility this doesn't happen?

If we know for certain we're getting an `obj::ratio` back, we should just use `make_box` to build it instead.

Reply:
yes, it should always be a ratio since we made sure a ratio_data is created only when its denominator does not simplify to 1. updated.